pkg/asset/rhcos/image.go

```diff
@@ -158,6 +159,11 @@ func osImage(config *types.InstallConfig) (string, error) {
 			return rhcos.FindArtifactURL(a)
 		}
 		return "", fmt.Errorf("%s: No vmware build found", st.FormatPrefix(archName))
+	case alibabacloud.Name:
```

Comment:
This should probably mirror AWS. We will be providing an rhcos image specifically built for Alibaba. This image should be available and the images can be found here.  https://github.com/openshift/installer/blob/master/data/data/rhcos-amd64.json 
@fabianofranz @ashcrow 
What are the next steps to getting the rhcos into this list? The 1-off image was tested and worked great.

Reply:
In terms of time frame I'm not sure when this will occur but once the pipeline is updated to produce this image and is uploading it we will run [this script](https://github.com/openshift/installer/blob/master/hack/update-rhcos-bootimage.py) to include it in the installer json location and open a PR.

cc @miabbott

Reply:
Next steps:

1.  Start publishing Aliyun images from RHCOS pipeline
2.  Make sure that coreos/stream-metadata-go has support for Aliyun
3.  Run `plume cosa2stream` to update `data/data/rhcos-stream.json`
4.  Run `hack/update-rhcos-bootimage.py`
5.  Verify that changes include `aliyun` artifact
6.  Submit PR to update `openshift/installer`

Reply:
@miabbott 

> Start publishing Aliyun images from RHCOS pipeline

I believe the work here is encompassed in the epic here: https://issues.redhat.com/browse/GRPA-3312. We need to decide the location of where the images need to be placed and provide Micah's team with the credentials.
cc @fabianofranz @coverprice @gauravsingh85 
> Make sure that coreos/stream-metadata-go has support for Aliyun

https://github.com/coreos/stream-metadata-go/blob/main/release/release.go#L55  Is this sufficient?
> Run plume cosa2stream to update data/data/rhcos-stream.json

This requires 1. to be completed.
> Verify that changes include aliyun artifact

This requires 1. to be completed.
> Submit PR to update openshift/installer

This is something that we will do as soon as we can.

Reply:
Alibaba will be based on same model as AWS . Each region will have its own RHCOS image . The work will begin in first sprint og 4.10 . If this is a blocker to accept the and merge the PR then please let me know

Reply:
@bd233 @dongchen126 
Please update this section to match the AWS section above.
```
https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/image.go#L85-L99
```
Update this code to match Alibaba's API calls. This will also require adding an ImageID property to the Alibaba platform code here: https://github.com/openshift/installer/pull/5018/files#diff-66a5c76a98269d93a28c9e80eeff0c769240dd4621fe1691e953a7e3b07c5c36

Reply:
> I believe the work here is encompassed in the epic here: https://issues.redhat.com/browse/GRPA-3312. We need to decide the location of where the images need to be placed and provide Micah's team with the credentials.
> cc @fabianofranz @coverprice @gauravsingh85

Yup, that looks correct!  We have credentials for our devel pipeline, but need to sort out the final location for production releases + credentials for ART (as noted).

> > Make sure that coreos/stream-metadata-go has support for Aliyun
> 
> https://github.com/coreos/stream-metadata-go/blob/main/release/release.go#L55 Is this sufficient?

Looks like it!  Once we start pushing out artifacts from the devel pipeline we can verify that support is all that is needed.

> > Run plume cosa2stream to update data/data/rhcos-stream.json
> 
> This requires 1. to be completed.
> 
> > Verify that changes include aliyun artifact
> 
> This requires 1. to be completed.
> 
> > Submit PR to update openshift/installer
> 
> This is something that we will do as soon as we can.

Apologies for being overly specific in the todo items; once 1 + 2 are knocked off from the list, the rest should be straight forward.

Reply:
@miabbott Do we have an update to when the Alibaba images will be updated in the corresponding streams?